src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.ts

```diff
@@ -42,6 +42,9 @@ export class SideNavContentComponent implements OnInit {
 
   public quickMapsForm: FormGroup;
 
+  public sideNaveToggleLock = new FormControl(false);
+  public lockSideNav = false;
```

Comment:
I don't think that you need both of these.  The formcontrol is probably the right way to go.
I'd suggest that you:
- change the spelling to "sideNavToggleLock" :-)
- Remove the "lockSideNav" variable.
- Remove the "sideNavLockChange" function.
- In the "minimiseSideNav" "if" function check "this.sideNavToggleLock.value" instead.